controllers/childrenController.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../models");
+
+// Defining methods for the childController
+module.exports = {
+  //find all children related to requestor
+  findAllByChild: function(req, res) {
+    db.Job
+      .findAll({where:{AssignedToEmail:req.body.email}})
+      .then(dbModel => res.json(dbModel))
```

Comment:
These need to be giving back a status, message, and data probably. dbModel is very vague.